understanding/20/use-of-color.html

```diff
@@ -154,21 +154,34 @@ <h2>Benefits of Use of Color</h2>
    <section id="examples">
       <h2>Examples of Use of Color</h2>
       
-      <dl>
-         <dt>A form that uses color and text to indicate required fields</dt>
-         <dd>A form contains both required and optional fields. Instructions at the top of the
+      <h3>A form that uses color and text to indicate required fields</h3>
+         <p>A form contains both required and optional fields. Instructions at the top of the
                form explain that required fields are labeled with red text and also with an icon.
                Users who cannot perceive the difference between the optional field labels and the
                red labels for the required fields will still be able to see the icon next to the
-               red labels.</dd>
-         <dt>An examination</dt>
-         <dd>Students view an SVG image of a chemical compound and identify the chemical elements
+               red labels.</p>
+     
+         <h3>An examination</h3>
+         <p>Students view an SVG image of a chemical compound and identify the chemical elements
                present based <strong>both</strong> on the colors used, as well as numbers next to each
                element. A legend shows the color and number for each type of element. Sighted users who
                cannot perceive all the color differences can still understand the image by relying on
-               the numbers.</dd>
-      </dl>
-      
+               the numbers.</p>
+     
+         <h3>Focus on links</h3>
```

Comment:
I find the entire description of Focus on links extremely hard to follow, and I do not find the visual examples really help to guide the user to an understanding of whether this is demonstrating a pass or fail situation. This is made more problematic in that some of the items in this new section are passes and some are fails.
If you look at Non-Text Contrast's understanding document, the figure captions explicitly call out whether the example is a pass or fail (in fact, my recollection is we had "Failure example" right at the start of these at one point). The short clear figure captions in https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html may help guide a better way to approach these.

Reply:
Since the links one is the only failure example, maybe we should make them all pass examples? We already have a failure technique with links; maybe we should add a visual to that one, or expand it?